docs/reference/config.rst

```diff
@@ -167,20 +167,38 @@ Defaults to ``yes``.
 list_format_item
 ~~~~~~~~~~~~~~~~
 
+Deprecated option, replaced by :ref:`format_item`.
+
+.. _list_format_album:
+
+list_format_album
+~~~~~~~~~~~~~~~~~
+
+Deprecated option, replaced by :ref:`format_album`.
```

Comment:
Ah, got it—that would be unfortunate. Yes, let's remove the sections. Where we mention the old option, we can just not use a link (i.e., just use double or single backticks).